### DISCUSSION TITLE
A Slack group for Graphql-WS

### DISCUSSION THREAD

**Discussion by 97amarnathk** [enisdenjo/graphql-ws#162] [General]
Several other projects like `KafkaJS` have their own Slack groups.

It would be a great idea to have a slack group for Graphql WS as well. Since graphql subscriptions are a relatively new topic with several different use cases and caveats.

This would make discussions much faster.

**Comment by n1ru4l** [comment]
There already is a [GraphQL Community Discord](https://discord.gg/qf54FMt8), [GraphQL Slack](https://graphql-slack.herokuapp.com/) and [The Guild - Community Discord](https://discord.gg/sqUQDXuE)

A channel for explicitly graphql-ws could be created in any of those.

**Reply by enisdenjo** [reply]
Yeah, we should just use those channels. I am available on all of them - but am mainly on GraphQL Slack.

**Reply by n1ru4l** [reply]
I created the graphql-ws https://graphql.slack.com/archives/C01V41494LS channel
